runelite-client/src/test/java/net/runelite/client/plugins/PluginManagerTest.java

```diff
@@ -95,6 +98,37 @@ public void before() throws IOException
 			if (pluginDescriptor != null)
 			{
 				pluginClasses.add(clazz);
+				continue;
+			}
+
+			// Ensure plugin configurations do not repeat keyNames
+			if (Config.class.isAssignableFrom(clazz))
```

Comment:
You should just build list of config classes here and then make separate test, `@Before` is for preparing environment for tests, not for testing stuff

Reply:
I've done just that. :+1: